assets/src/containers/FilesAccessed.js

```diff
@@ -189,9 +200,23 @@ function FilesAccessed (props) {
               onWeekChange={onWeekChangeHandler}
             /> : ''}
             <div className={classes.formController}>
-              <p>File accessed from
+              <FormControl className={classes.formControl}>
+                <Select
+                  value={fileFilter}
+                  onChange={onChangeFileHandler}
+                  inputProps={{
+                    name: 'file',
+                    id: 'file-type',
+                  }}
+                >
+                  <MenuItem value="all_files">All</MenuItem>
```

Comment:
Can we make this list configurable from settings file? Not only choice values, but also labels.

Reply:
I will try to fix this